### PR TITLE
CVSB-12157- config.yml updated for dynamo db config and max retries

### PR DESCRIPTION
**Pull request by m-ujaffer**
https://jira.dvsacloud.uk/browse/CVSB-12157

### CODE REVIEW

File: src/config/config.yml [hunk 1]
```diff
@@ -9,16 +9,25 @@ dynamodb:
...
     params:
       region: localhost
       endpoint: http://localhost:8000
+      maxRetries: 1
+      httpOptions:
+        timeout: 1000
     table: cvs-local-global-test-number
   remote:
-    params: {}
+    params:
```
Comment (agoncearenco) [inline src/config/config.yml]: Is this config enough? Or does it require a DevOps change also?

Comment (KHodderDVSA) [reply src/config/config.yml]: Config is used by the app directly. Shouldn't require a devops change

Comment (KHodderDVSA) [reply src/config/config.yml]: Oh, DB changes - Yes, this would need to be terraformed.

**Comment by m-ujaffer** [comment]
> This does nothing on its own - needs to be pushed into Terraform.

I added console log and ran on AWS. Cloud watch indicates that params are passed to DynamoDB Client:

`2020-03-04T11:49:22.080Z	1a11fbdb-2d23-4587-b8d6-8f4baa8f968c	INFO	config for DynamoDB Client:  { maxRetries: 1, httpOptions: { timeout: 1000 } }`

**Pull request merged**